examples/plotting/great_arc_example.py

```diff
@@ -7,7 +7,6 @@
 Sun, and to extract intensity values along that arc.
 """
 import matplotlib.pyplot as plt
-import numpy as np
 
 import astropy.units as u
 from astropy.coordinates import SkyCoord
```

Comment:
```suggestion
from astropy.coordinates import SkyCoord
from astropy.visualization import quantity_support
```

Reply:
I've put in quantity support, but left the manual axes labelling in because I think it adds useful information to the plot

Reply:
That makes sense. But then I don't think importing quantity_support is needed.